prow/cmd/phaino/local.go

```diff
@@ -418,7 +418,7 @@ func (opts *options) resolveRefs(ctx context.Context, volumeMounts map[string]st
 		return scanln(ctx)
 	}
 
-	goSrcPath := filepath.Join(opts.gopath, "src")
+	goSrcPath := filepath.Join(opts.codeMountPath, "src")
```

Comment:
It'll become confusing to append `src` and use it as the final mount path.. Probably better to also ask users to add `src` as part of the --code-mount-path?

Reply:
good point, updated.